cmd/podman/pull.go

```diff
@@ -64,11 +64,11 @@ specified, the image with the 'latest' tag (if it exists) is pulled
 // pullCmd gets the data from the command line and calls pullImage
 // to copy an image from a registry to a local machine
 func pullCmd(c *cli.Context) error {
-	runtime, err := libpodruntime.GetRuntime(c)
+	localRuntime, err := adapter.GetRuntime(c)
```

Comment:
Is runtime more descriptive than localRuntime? Since this could be the remote runtime?  Or am I wrong?